app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
 class PagesController < ApplicationController
-  def start; end
+  def helpdesk_request_submitted
+    @trn_request = TrnRequest.find(session[:trn_request_id])
+  end
```

Comment:
Should this live in `PagesController`? My thought was that this controller would be for static pages.

Reply:
I wasn't sure if we would need a show action for the `TrnRequest` resource, so figured it would be easiest for now to use `PagesController` to get the desired URL and defer the decision to a later point when we know more.